package.json

```
@@ -24,19 +24,19 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^10.1.0",
+    "@theforeman/vendor": "^10.1.4",
     "graphql-tag": "^2.11.0",
     "intl": "~1.2.5",
     "jed": "^1.1.1",
     "react-intl": "^2.8.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^10.1.0",
-    "@theforeman/eslint-plugin-foreman": "^10.1.0",
-    "@theforeman/stories": "^10.1.0",
-    "@theforeman/test": "^10.1.0",
-    "@theforeman/vendor-dev": "^10.1.0",
+    "@babel/core": "7.17.10",
```

```suggestion
    "@babel/core": ">= 7.7.0 < 7.17.12",
```

this will do what you want, w/o freaking out packaging